model/models.py

```diff
@@ -20,14 +20,26 @@
 # pytype: disable=wrong-keyword-args
 # pytype: disable=attribute-error
 
-from typing import Callable, Any, Optional
+from functools import partial
+from typing import (Any, Callable, Tuple, Optional)
```

Comment:
Nit: parentheses can be removed.
```suggestion
from typing import Any, Callable, Tuple, Optional
```